Sources/OutlineView/OutlineView.swift

```diff
@@ -30,6 +44,109 @@ where Data.Element: Identifiable {
+    }
+
+    public func updateNSViewController(
+        _ outlineController: OutlineViewController<Data, Drop>,
+        context: Context
+    ) {
+        outlineController.updateData(newValue: data)
+        outlineController.changeSelectedItem(to: selection)
+        outlineController.setRowSeparator(visibility: separatorVisibility)
+        outlineController.setRowSeparator(color: separatorColor)
+        outlineController.setDragSourceWriter(dragDataSource)
+        outlineController.setDropReceiver(dropReceiver)
+        outlineController.setAcceptedDragTypes(acceptedDropTypes)
+    }
+}
+
+// MARK: - Modifiers
+
+@available(macOS 10.15, *)
+public extension OutlineView {
+    /// Sets the style for the `OutlineView`.
+    @available(macOS 11.0, *)
+    func outlineViewStyle(_ style: NSOutlineView.Style) -> Self {
+        var mutableSelf = self
+        mutableSelf.style = style
+        return mutableSelf
+    }
+
+    /// Sets the width of the indentation per level for the `OutlineView`.
+    func outlineViewIndentation(_ width: CGFloat) -> Self {
+        var mutableSelf = self
+        mutableSelf.indentation = width
+        return mutableSelf
+    }
+
+    /// Sets the visibility of the separator between rows of this outline view.
+    func rowSeparator(_ visibility: SeparatorVisibility) -> Self {
+        var mutableSelf = self
+        mutableSelf.separatorVisibility = visibility
+        return mutableSelf
+    }
+
+    /// Sets the color of the separator between rows of this outline view.
+    /// The default color for the separator is `NSColor.separatorColor`.
+    func rowSeparatorColor(_ color: NSColor) -> Self {
+        var mutableSelf = self
+        mutableSelf.separatorColor = color
+        return mutableSelf
+    }
+
+    /// Adds a drop receiver to the OutlineView, allowing it to react to drag
+    /// and drop operations.
+    /// - Parameters
+    ///   - receiver: An object conforming to `DropReceiver` that handles
+    ///   - acceptedTypes: An array of `PasteboardType`s that the `DropReceiver` is able to read
+    ///   drag-and-drop receiving for the OutlineView
+    func onDrop(of acceptedTypes: [NSPasteboard.PasteboardType], receiver: Drop) -> Self {
+        var mutableSelf = self
+        mutableSelf.dropReceiver = receiver
+        mutableSelf.acceptedDropTypes = acceptedTypes
+        return mutableSelf
+    }
+    
+    /// Enables dragging of rows from the OutlineView by setting the `DragSourceWriter`
+    /// of the OutlineView
+    /// - Parameter writer: A closure that takes the `Data.Element` from a given row of
+    /// the OutlineView, and returns an optional `NSPasteboardItem` with data about the
+    /// item to be dragged. If nil is returned, that row can not be dragged.
+    ///
+    /// The simplest way to create the data for the pasteboard item is to initialize
+    /// the `NSPasteboardItem` and then calling `setData(_:forType:)` or other types
+    /// of `set` functions.
```

Comment:
As a convention, the description should precede the parameters. (Just based on the order in which they appear in the Quick Help)